internal/process/defunct_processes_test.go

```diff
@@ -0,0 +1,12 @@
+package process
+
+import "testing"
+
+func TestDefunctProcessesForPath(t *testing.T) {
+	defunctCount, err := DefunctProcessesForPath("./testing/proc")
+	if err != nil {
+		t.Errorf("%v", err)
+	} else if defunctCount != 7 {
+		t.Errorf("DefunctProcessesForPath returned %d; should return 7.", defunctCount)
+	}
+}
```

Comment:
While this is fine, let's use the ginkgo test framework like we do in other tests within this repository. This means, we add a `suite_test.go` like this: https://github.com/cri-o/cri-o/blob/master/internal/config/apparmor/suite_test.go

Then we modify the `defunct_processes_test.go` to this basic skeleton:
```go
package process_test

import (
	. "github.com/onsi/ginkgo"
	. "github.com/onsi/gomega"

	"github.com/cri-o/cri-o/internal/process"
)

// The actual test suite
var _ = t.Describe("Process", func() {
	t.Describe("DefunctProcessesForPath", func() {
		It("should succeed", func() {
			defunctCount, err := process.DefunctProcessesForPath("./testing/proc")

			Expect(err).To(BeNil())
			Expect(defunctCount).To(Equal(7))
		})
	})
})
```
Let's also add a tests for the `err != nil` cases.